### ISSUE TITLE
Disconnections of the AAW at specific locations.

### ISSUE THREAD

**Issue by Dragoshiz** [cpebit/aawireless-issues#50]
In some countries the WiFi channel we use now (40) might be not ideal cause of interference, AAWireless might disconnect. We need to provide an option the change the channel and/or channel width.

More info available here: https://cpeb.freshdesk.com/a/tickets/2487

**Comment by Snirpo** [comment]
We can optimize the 2.4Ghz setting when 5Ghz does not work correctly for some regions.

David: 

I think I found a 2.4GHz optimisation. I enabled short-gi-20, PSMP, and require HT and did my very scientific go to maps and zoom in and out as quickly as I can. It was super smooth. Like when on 5GHz. I need to do a drive to test properly but I am done with driving for today. If you have a HU and want to test yourself add these lines to the 2.4GHz configuration:
ht_capab=[SHORT-GI-20][PSMP]
require_ht=1
